src/modules/UI/scenes/Plugins/api.js

```diff
@@ -66,13 +69,22 @@ export class PluginBridge {
     return Promise.reject(new Error('not implemented'))
   }
 
-  chooseWallet (): Promise<any> {
+  chooseWallet (obj: { cbid: string, func: string, id: string, currencyCode: string }): Promise<any> {
+    this.context.chooseWallet(obj.id, obj.currencyCode)
+    return Promise.resolve(null)
+  }
+
+  changeWallet (): Promise<any> {
     this.context.toggleWalletList()
     return Promise.resolve(null)
   }
 
+  getAPIKey (): Promise<string> {
+    const key: string = ENV.BIT_REFILL_API_KEY
```

Comment:
Please rename this, we may have tons of plugins which means `getApiKey` can't only be relevant for BitRefill, etc.

Reply:
There are already functions for getting a value from the GUI. For reference, look at how the Simplex plugin was doing this.

Reply:
@kylanhurt changed to a better naming convention that will get the api key from the plugin name in an object of all the plugin api keys

Reply:
I'm not looking at the code, but in @teneighty 's docs here

https://github.com/EdgeApp/edge-docs/blob/master/source/includes/_plugins.md

He references a `Config` object where you can do a get() call specifically for API_KEYs. Was that not implemented?

Reply:
<img width="922" alt="screen shot 2019-01-05 at 14 18 48" src="https://user-images.githubusercontent.com/6079354/50729806-dbf76900-10f4-11e9-8932-9f9f3508332b.png">

Reply:
Ok, refactored to use that and it is working. New method has been removed.